utils/utils.go

```diff
@@ -171,6 +175,16 @@ func GetIdentifiers(args *skel.CmdArgs) (workloadID string, orchestratorID strin
 	return workloadID, orchestratorID, nil
 }
 
+func GetHandleID(netName string, containerID string) (string, error) {
```

Comment:
It would be really useful to include a log in this function which correlates the netname, containerID, and the generated handle ID.

Do we get that anywhere else? Need to make sure that shows up both in ADD and DEL code paths.

Reply:
Added a context logger here, not sure if it is exactly what you had in mind but it should get the idea across